src/test/java/com/ontotext/kafka/Utils.java

```diff
@@ -0,0 +1,68 @@
+package com.ontotext.kafka;
+
+import com.ontotext.kafka.mocks.DummyRepository;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.eclipse.rdf4j.repository.Repository;
+import org.eclipse.rdf4j.rio.RDFFormat;
+
+import java.io.Reader;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Queue;
+import java.util.function.Supplier;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class Utils {
+
```

Comment:
Util classes such as this one have private constructors in order to avoid any unnecessary object creation by other developers who might use this class.